packages/react-storage/src/components/StorageBrowser/views/LocationActionView/UploadControls.tsx

```diff
@@ -296,7 +299,11 @@ export const UploadControls = (): JSX.Element => {
   const disableSelectFiles = hasStarted || hasCompleted;
 
   return (
-    <>
+    <DropZoneElement
```

Comment:
Drag and drop should be restricted to the `Table`

Reply:
Moved to `Table`